config/config.go

```diff
@@ -140,13 +176,40 @@ func (cfg Config) Validate() error { //nolint:gocognit
 		inputErrors = append(inputErrors, fmt.Errorf("-globalTimeout: must be > 0, we got %d", cfg.RunnerOptions.GlobalTimeout))
 	}
 
+	if !reflect.ValueOf(cfg.Request.Body).IsZero() {
+		if len(cfg.Request.Body.Content) == 0 {
+			inputErrors = append(inputErrors, fmt.Errorf("-bodyContent: you must provide a value if you have added a bodyType"))
+		}
+		if reflect.ValueOf(cfg.Request.Body.Type).IsZero() {
```

Comment:
Let's avoid calls to `reflect` when we can
```suggestion
		if cfg.Request.Body.Type == "" {
```